src/components/Comments/ContainerComment.js

```diff
@@ -227,7 +227,7 @@ export const styles = StyleSheet.create({
   },
   containerComment: {
     borderLeftWidth: 1,
-    borderLeftColor: '#C4C4C4',
+    borderLeftColor: COLORS.gray1,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The borderLeftColor property in the style object has been updated to use `COLORS.gray1` instead of a hard-coded color value. This change improves maintainability by centralizing color definitions, making it easier to update colors consistently across the application in the future.
```diff
-    borderLeftColor: '#C4C4C4',
+    borderLeftColor: COLORS.gray1,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->